kas/anchor.go

```diff
@@ -66,6 +68,86 @@ func NewKASAnchor(kasConfig *KASConfig, db AnchorDB, bc BlockChain) *Anchor {
 	}
 }
 
+// AnchorPeriodicBlock periodically anchor blocks to KAS.
+// if given block is invalid, it does nothing.
+func (anchor *Anchor) AnchorPeriodicBlock(block *types.Block) {
+	if !anchor.kasConfig.Anchor {
+		return
+	}
+
+	if block == nil {
+		logger.Error("KAS Anchor : failed to get a block form the event")
```

Comment:
```suggestion
		logger.Error("KAS Anchor : failed to get a block from the event")
```

Reply:
Log message is a bit weird since there is no event in the current function call.

Reply:
reflected